### ISSUE TITLE
Blank Screen which black, white or brown on load

### ISSUE THREAD

**Issue by cihanuygun** [a85/POSTMan-Chrome-Extension#4958]
Please read through the [guidelines](https://github.com/postmanlabs/postman-app-support#guidelines-for-reporting-issues) before creating a new issue.

**Describe the bug**
Postman is not loading since v6.0.10

**To Reproduce**
Run postman from shortcut, cmd or directly from exe.

**Expected behavior**
Simply just run.

**Screenshots**
Initial run: 
![image](https://user-images.githubusercontent.com/5497248/43647695-201e2bce-9741-11e8-911c-a4de35a271b5.png) 
Resizing window: 
![image](https://user-images.githubusercontent.com/5497248/43647807-654a5fce-9741-11e8-90d9-e294614419fa.png)
I can find menus blindly but they are not functional see next screenshot:
![image](https://user-images.githubusercontent.com/5497248/43647873-93368926-9741-11e8-9172-fb4cf0889560.png)
Developer Tools (Current View): 
![image](https://user-images.githubusercontent.com/5497248/43647937-b8aac528-9741-11e8-8a7e-61771e69b698.png)

**CMD Log**
```
ORM initialization successful
App events bus { name: 'booted', namespace: 'shared' }
App updater event { name: 'checkForElectronVersionUpdated',
  namespace: 'appUpdate' }
App events bus { name: 'booted', namespace: 'requester' }
Closed Window (id: 3)
```

**App information:**
 - Native App
 - Postman Version 6.2.3 x64
 - OS: Windows 8.1 x64 with latest updates
 - No cloud just running it standalone without an account

**Additional context**
This problem showed up since the version 6.0.10 still without updating postman this version works well.

**Comment by sivcan** [comment]
@cihanuygun Try this: https://github.com/postmanlabs/postman-app-support/issues/4703#issuecomment-397517447

**Comment by numaanashraf** [comment]
Duplicate of #4703

**Comment by cihanuygun** [comment]
Thank you @sivcan that solved my GUI problem and that time #4897 showed up and when i applied the workaround @kamalaknn provided now postman running properly. Thanks.
